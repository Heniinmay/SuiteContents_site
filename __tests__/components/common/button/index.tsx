import { render } from '@testing-library/react'
import Button from 'src/components/common/button'

describe('Button', () => {
  it('renders a button', () => {
    const { container } = render(<Button />)

    expect(container).toMatchSnapshot()
  })
})
