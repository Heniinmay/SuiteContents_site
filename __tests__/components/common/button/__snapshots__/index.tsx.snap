// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders a button 1`] = `
<div>
  <button
    class="css-hhi3rx"
  />
</div>
`;
